src/api/v1/follow.ts

```diff
@@ -139,10 +139,10 @@ router.get('/', async (request: express.Request, response: express.Response) =>
   }
 
   const follows = result.map(data => data.dataValues[target]);
-
+  
   response.json({
     follows: follows,
-    pageToken: follows.length > 0 ? follows[follows.length - 1].id : null
+    pageToken: follows.length > 0 ? result[result.length-1].dataValues.id : parseInt(request.query.pageToken as string)

```

Comment:
```suggestion
    pageToken: follows.length > 0 ? result[result.length-1].dataValues.id : lastFollowId
```
위 코드가 깔끔할 것 같긴한데, null에서 바꾼 이유가 있을까요? parseInt의 결과가 NaN이 나올 수 도 있을것 같아서요.